packages/canon/docs/Iconography.mdx

```diff
@@ -0,0 +1,20 @@
+import { Unstyled, Source } from '@storybook/blocks';
+import { Title, Text, IconLibrary } from './components';
+
+<Unstyled>
+
+<Title type="h1">Iconography</Title>
+
+<Text>
+  All our default icons are provided by [Remix Icon](https://remixicon.com/). We
```

Comment:
how come you switched?

Reply:
Lucide was quite unbalanced. Daniele and Dan tested a couple of libraries and Remix was the one that worked the best so far. In term of license they are open source under an Apache license so we are good.